parity/run.rs

```diff
@@ -115,7 +117,20 @@ pub fn execute(cmd: RunCmd) -> Result<(), String> {
 
 	// display info about used pruning algorithm
 	info!("Starting {}", Colour::White.bold().paint(version()));
-	info!("Using state DB journalling strategy {}", Colour::White.bold().paint(algorithm.as_str()));
+	info!("State DB configuation: {}{}{}",
+		Colour::White.bold().paint(algorithm.as_str()),
+		match cmd.fat_db {
+			Switch::On if algorithm == Algorithm::Archive => format!("{}", Colour::White.bold().paint(" +Fat")), 
+			Switch::On => die("Fat DB is not supported with the chosen pruning option. Please rerun with `--pruning=archive`"),
+			Switch::Auto => die("--fat-db auto is not supported."),
+			Switch::Off => "".to_owned(),
+		},
+		match cmd.tracing {
+			Switch::On => format!("{}", Colour::White.bold().paint(" +Trace")),
+			Switch::Auto => format!("{}", Colour::White.bold().paint(" +?Trace")),
```

Comment:
it can be replaced with shorter: `Colour::White.bold().paint(" +?Trace").to_string()`